_sass/blocks/_chart-list.scss

```diff
@@ -0,0 +1,73 @@
+.chart-list {
+  @include font-size(0.8);
+  @include outer-container();
+
+  .chart-title {
+    border-bottom: 1px solid $mid-gray;
+    font-size: inherit;
+    font-weight: $weight-book;
+    margin-bottom: $base-padding;
+    text-transform: uppercase;
+  }
+
+  .chart-list-intro {
+    @include span-columns(8);
+  }
+
+  .chart-item {
+    @include span-columns(4);
+
+    border-top: none;
+    padding-top: 0;
+    margin-top: 0;
+  }
+
+  .full-width {
+    @include span-columns(12);
+    @include omega();
+  }
+
+  .row-container {
+    @include outer-container();
+
+    > * {
+      @include span-columns(4);
+    }
+
+    > :nth-child(3) {
+      @include omega();
+    }
+  }
+
+  &.has-intro {
+    .chart-item:first-of-type,
+    .chart-item:nth-of-type(3n + 1) {
+      @include omega();
+    }
+  }
+
+  &:not(.has-intro) {
+    .chart-item:nth-of-type(3n) {
+      @include omega();
+    }
+  }
+
+  figure {
+    margin-top: 0;
+  }
+
+  figcaption {
+    font-weight: $weight-bold;
+    display: flex;
+    padding: 5px; // FIXME this is an approximation
```

Comment:
will `$base-padding-lite` work here? It generally ends up computing to around 5px